### ISSUE TITLE
[EN][Quest][Icecrown] Borrowed Technology

### ISSUE THREAD

**Issue by Malanie** [Atlantiss/BugTracker#8964]
**Description**: After you assembley the Dummy and the Drake approaches you dont get a count for the quest

**Current behaviour**: you can collect the 3 items needed to create the dummy, u can use the provided item to create a dummy, a drakes comes to the dummy and flys off

**Expected behaviour**: after that you should get a count for the quest

1. go to icecrown, find the airship and get the quest from  Chief Engineer Boltwrench
2. head to the quest area collect the items and create a dummy
3. see the drake come and go without anything to happen

**Comment by Piklo** [comment]
https://github.com/Atlantiss/BugTracker#how-to-use
[wowhead](http://www.wowhead.com/) or [cata.openwow.com](http://cata.openwow.com/) links

**Comment by deadspear** [comment]
Yeah all the parts of this quest work, except the Frost Wyrm that picks up the Dummy should die.
https://www.youtube.com/watch?v=TexTwmdrOEI
http://www.wowhead.com/quest=13291/borrowed-technology
